app/league/config_utils.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+"""Helper utility for managing application settings."""
+import json
+from os.path import exists, getsize, join
+
+from flask import current_app
+
+
+def update_config_file(new_config):
```

Comment:
one comment: `before_first_request_funcs` fires too late to be useful in config, particularly if we decide to add more to the admin panel.